spec.bs

```diff
@@ -1731,3 +1773,212 @@ specification is printed below:
   /fenced-frame/cspee.https.html
   /fenced-frame/embedder-csp-not-propagate.https.html
 </wpt>
+
+<h3 id=permissions-policy-changes>Permissions Policies</h3>
+
+*This introductory sub-section is non-normative.*
+
+The [=policy-controlled features=] available to {{Document}}s inside of a <{fencedframe}> are
+determined exclusively by the {{FencedFrameConfig}} that the <{fencedframe}> navigates to.
```

Comment:
This is true right now, but maybe not true with future changes (or default mode, depending on how we spec it).

Reply:
Right, thanks. For now I'm just worried about spec'ing opaque stuff basically.